umrr_ros2_driver/CMakeLists.txt

```diff
@@ -30,8 +30,11 @@ endif()
 
 find_package(ament_cmake_auto REQUIRED)
 find_package(Threads REQUIRED)
+find_package(umrr_ros2_msgs REQUIRED)
 
 set(ENABLE_CJSON_UNINSTALL OFF CACHE BOOL "Enable creating uninstall target")
+set(FASTRTPS_INCLUDE_DIR "/opt/ros/foxy/include")
+set(FastRTPTS_LIBRARY_RELEASE "/opt/ros/foxy/lib/linfastrtps.so")
```

Comment:
This binds the package to a specific DDS vendor (FastRTPS) which is bad practice. ROS2 allows you to switch DDS vendors for a reason.